policy-engine/src/openapiv3.json

```diff
@@ -0,0 +1,89 @@
+{
+    "openapi": "3.0.2",
+    "info": {
+        "version": "0.0.0",
+        "title": "OpenShift Cincinnati Policy-Engine",
+        "license": {
+            "name": "Apache2"
+        },
+        "contact": {}
+    },
+    "servers": [],
+    "paths": {
+        "{{path_prefix}}/v1/graph": {
+            "get": {
+                "summary": "Get the update graph",
+                "operationId": "getGraph",
+                "parameters": [],
+                "responses": {
+                    "200": {
+                        "description": "An update graph",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Graph"
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "unexpected error",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "schemas": {
+            "Graph": {
+                "type": "array",
```

Comment:
I think this is wrong. A graph response is a top-level object containing two fields, `nodes` and `edges`, both of which are arrays with a homogeneous type.